rnote-cli/src/cli.rs

```diff
@@ -33,13 +33,17 @@ pub(crate) enum Commands {
     },
     /// Exports the Rnote file and saves it in the output file.{n}
     /// The export format is recognized from the file extension of the output file.{n}
+    /// When using --output-format, the same file name is used with the extension changed.{n}
     /// Currently `.svg`, `.xopp` and `.pdf` are supported.
     Export {
         /// the rnote save file
```

Comment:
file -> files. Maybe add a doc comment to explain the valid combinations here

Reply:
Yes, I have now also used clap to handle in a better way that they are required and conflicting between each other.

Reply:
Nice, I like it!